base/collect_database_test.go

```diff
@@ -33,32 +29,6 @@ type collectDatabaseTemplateInput struct {
 	GUID     string `json:"GUID" yaml:"GUID"`
 }
 
-// has to be a map[string]string in order to do input checks in template
-// func executeTemplate(inputs interface{}, templatePath string, writePath string) error {
-func executeTemplate(inputs map[string]interface{}, templatePath string, writePath string) error {
```

Comment:
Unfortunately, by simplifying the DB task test (no longer generates the metrics file from a template), I have to put the metrics file in `base/` because of how the DB task works. It looks for the metrics file in the current directory and in the context of the test, it is in `base/`. Thoughts?

Reply:
`base/test-ce.metrics.yaml`

Reply:
I feel we can cleanup the tests a bit to address this in the short term, and cleanup the file driver and kubedriver to address this in the next PR.

Reply:
Perhaps, we should put this file under `testdata/metrics` folder, do a `os.Chdir(t.TempDir())` within the unit tests, and then copy the `base/test-ce.metrics.yaml` file into the tempDir?

```go
// approximate code
destDir := t.TempDir()
os.Chdir(destDir)
src := base.CompletePath("../testdata", "metrics/test-ce.metrics.yaml")
io.Copy(src, base.CompletePath(destDir, metricYamlFileName))
// rest of the test goes here... 
```